i18n/ads-language-pack-de/translations/extensions/schema-compare.i18n.json

```diff
@@ -9,154 +9,10 @@
-			"SchemaCompare.Contracts": "Verträge",
-			"SchemaCompare.CreateNewDatabase": "Neue Datenbank erstellen",
-			"SchemaCompare.Credentials": "Anmeldeinformationen",
-			"SchemaCompare.CryptographicProviders": "Kryptografieanbieter",
-			"SchemaCompare.DatabaseAuditSpecifications": "Spezifikationen für Datenbanküberwachung",
-			"SchemaCompare.DatabaseEncryptionKeys": "Verschlüsselungsschlüssel für Datenbank",
-			"SchemaCompare.DatabaseOptions": "Datenbankoptionen",
-			"SchemaCompare.DatabaseRoles": "Datenbankrollen",
-			"SchemaCompare.DatabaseScopedCredentials": "Datenbankweit gültige Anmeldeinformationen",
-			"SchemaCompare.DatabaseTriggers": "Datenbanktrigger",
-			"SchemaCompare.Defaults": "Standardwerte",
-			"SchemaCompare.DeployDatabaseInSingleUserMode": "Datenbank im Einzelbenutzermodus bereitstellen",
-			"SchemaCompare.DisableAndReenableDdlTriggers": "DDL-Trigger deaktivieren und erneut aktivieren",
-			"SchemaCompare.DoNotAlterChangeDataCaptureObjects": "Change Data Capture-Objekte nicht ändern",
-			"SchemaCompare.DoNotAlterReplicatedObjects": "Replizierte Objekte nicht ändern",
-			"SchemaCompare.DropConstraintsNotInSource": "Nicht in der Quelle enthaltene Einschränkungen löschen",
-			"SchemaCompare.DropDmlTriggersNotInSource": "Nicht in der Quelle enthaltene DML-Trigger löschen",
-			"SchemaCompare.DropExtendedPropertiesNotInSource": "Nicht in der Quelle enthaltene erweiterte Eigenschaften löschen",
-			"SchemaCompare.DropIndexesNotInSource": "Nicht in der Quelle enthaltene Indizes löschen",
-			"SchemaCompare.DropObjectsNotInSource": "Nicht in der Quelle enthaltene Objekte löschen",
-			"SchemaCompare.DropPermissionsNotInSource": "Nicht in der Quelle enthaltene Berechtigungen löschen",
-			"SchemaCompare.DropRoleMembersNotInSource": "Nicht in der Quelle enthaltene Rollenmitglieder löschen",
-			"SchemaCompare.DropStatisticsNotInSource": "Nicht in der Quelle enthaltene Statistiken löschen",
-			"SchemaCompare.Endpoints": "Endpunkte",
-			"SchemaCompare.ErrorMessages": "Fehlermeldungen",
-			"SchemaCompare.EventNotifications": "Ereignisbenachrichtigungen",
-			"SchemaCompare.EventSessions": "Ereignissitzungen",
-			"SchemaCompare.ExtendedProperties": "Erweiterte Eigenschaften",
-			"SchemaCompare.ExternalDataSources": "Externe Datenquellen",
-			"SchemaCompare.ExternalFileFormats": "Externe Dateiformate",
-			"SchemaCompare.ExternalStreamingJobs": "Externe Streamingaufträge",
-			"SchemaCompare.ExternalStreams": "Externe Streams",
-			"SchemaCompare.ExternalTables": "Externe Tabellen",
-			"SchemaCompare.FileTables": "Dateitabellen",
-			"SchemaCompare.Filegroups": "Dateigruppen",
-			"SchemaCompare.Files": "Dateien",
-			"SchemaCompare.FullTextCatalogs": "Volltextkataloge",
-			"SchemaCompare.FullTextStoplists": "Volltextstopplisten",
+			"OptionNotFoundWarningMessage": "Bezeichnung: {0} existiert nicht in der Option value name lookup",
```

Comment:
@smartguest @kisantia i saw a bunch of  deletions here, please confirm this is intended

Reply:
yup a bunch of hard coded DacFx options strings were removed now that they're coming from DacFx with the options changes Sai made

Reply:
good, just want to confirm.